integration/flink/gradle.properties

```diff
@@ -1,4 +1,4 @@
 jdk8.build=true
 version=0.14.0-SNAPSHOT
-flink.version=1.14.2
+flink.version=1.15.2
```

Comment:
1.15.2 here, and 1.15.1 in CI - which one do we want?

Reply:
good point! we want the latest but docker image is just for 1.15.1 (1.15.2 is not available) https://hub.docker.com/_/flink/tags?page=1&name=1.15

Will stick then to 1.15.1